lms/static/lms/fixtures/preview/course_preview.html

```diff
@@ -8,8 +8,12 @@
                         <option value="staff" selected>Staff</option>
                         <option value="student">Student</option>
                         <option value="specific student">Specific student</option>
-                        <option value="group-a" data-group-id="group-a">Student in Group A</option>
-                        <option value="group-b" data-group-id="group-b">Student in Group B</option>
+                        <option value="group-a" data-group-id="group-a" data-partition-id="test_partition_a_id">
+                            Student in Group A
```

Comment:
Nit: Update this fixture file too so it is more consistent with reality ("Learner").